packages/peregrine/lib/talons/WishlistPage/useCreateWishlist.js

```diff
@@ -16,18 +26,38 @@ export const useCreateWishlist = () => {
         setIsModalOpen(false);
     }, []);
 
-    const handleCreateList = useCallback(data => {
-        // TODO create list mutation is not available yet
-        // Will be handled in PWA-989
-        console.log('Creating wish list with data: ', data);
-        setIsModalOpen(false);
-    }, []);
+    const handleCreateList = useCallback(
+        async data => {
+            try {
+                await createWishlist({
+                    variables: {
+                        input: data
+                    },
+                    refetchQueries: [{ query: getCustomerWishlistsQuery }],
```

Comment:
Right implementation, wrong query 😄  this ended up adding another network round trip. Here's a screenshot of network requests with that mutation, which additionally showcases why we should be giving operations unique names as well.

![Screen Shot 2021-03-12 at 11 33 31 AM](https://user-images.githubusercontent.com/462953/110976960-2acd6880-8327-11eb-90f1-a384dffd2709.png)

This will be a little more involved, but totally worth it.

1. Migrate `wishlistPage.gql.js` to Peregrine. Update `useWishlistPage` talon to use the `defaultOperations` merge pattern.
2. `import { GET_CUSTOMER_WISHLIST } from './wishlistPage.gql'`
3. Use that query here
4. Remove `GET_CUSTOMER_WISHLISTS` from `createWishlist.gql.js`
5. Additionally, you can scale down the response of `CREATE_WISHLIST` to just `id`, since we expect to refetch here anyway

Reply:
Have not checked network(. Corrected it.

Thank you